### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by klaasnicolaas** [klaasnicolaas/home-assistant-glow#63] [Ideas]
Hacktoberfest is around the corner and so I thought it would be a good time to see what the Home Assistant Glow is still missing. After all, the project already has more than 200 stars ⭐! So let's see together what we could work on over the next month 😄 

**Suggestions**
- Improvements and additions to the documentation.
- Extra case models for any other type of boards.
- Etc.

Let me know below what you would like to see improved!

**Comment by raoulteeuwen** [comment]


**Reply by klaasnicolaas** [reply]
I think you're a bit in the wrong place in terms of feedback, the discussion here is specifically about the Home Assistant Glow and from your posts I'm not really getting anything that we can specifically improve on this device 😉 

But I can help you with that integration, but let's have that conversation elsewhere. Please contact me via Discord or email.
